include/runtime.h

```diff
@@ -77,6 +77,10 @@ namespace detail {
 
 		host_object_manager& get_host_object_manager() const;
 
+		bool is_dry_run() const { return cfg->is_dry_run().value_or(false); }
+
+		int get_command_count() const;
```

Comment:
I'd prefer not introducing to new members to the runtime's API for a single test case. Why not use `runtime_testspy` instead?